scss/styles.scss

```diff
@@ -1,10 +1,23 @@
+// variables
+$total-height: 768px;
+$total-width: 1366px;
+
 .container {
   align-items: center;
   background-color: $white-primary;
   display: flex;
-  height: 768px;
+  height: $total-height;
   justify-content: center;
-  width: 1366px;
+  width: $total-width;
+
+  @media(max-width: $breakpoint) {
+    display: flex;
```

Comment:
This media will inherit the same rules as it's parent so it's necessary to apply for example `display: flex` and `justify-content: center` again